reporters/kamon-prometheus/src/test/scala/kamon/prometheus/EmbeddedHttpServerSpec.scala

```diff
@@ -7,8 +7,9 @@ import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
 
 import java.io.FileNotFoundException
-import java.net.URL
+import java.net.{URL, URLConnection}
 import java.util.zip.GZIPInputStream
+import scala.jdk.CollectionConverters._
```

Comment:
This import does not work with earlier Scala versions, but I'm about to merge removing support for Scala 2.11 and 2.12. Probably there is no need to change this but just wait until #1387 gets merged